doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -9,8 +9,39 @@
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
-        "#3": "For a PostgreSQL database, use: \"postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8\"",
+        "#3": "For a MySQL database, use: \"mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7\"",
         "#4": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
+        "DATABASE_URL": "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
+    },
+    "dockerfile": [
+        "RUN apk add --no-cache --virtual .pgsql-deps postgresql-dev; \\",
+        "\tdocker-php-ext-install -j$(nproc) pdo_pgsql; \\",
+        "\tapk add --no-cache --virtual .pgsql-rundeps so:libpq.so.5; \\",
+        "\tapk del .pgsql-deps"
+    ],
+    "docker-compose": {
+        "docker-compose.yml": {
+            "services": [
+                "database:",
+                "  image: postgres:${POSTGRES_VERSION:-13}-alpine",
```

Comment:
@dunglas Not sure if I did already ask this 🙈 , but was there a specific reasons why this yaml files are instead of 4 spaces 2 spaces. And so are not similar to symfony config files?
Make it harder in handling .editorconfig files.

Reply:
4 spaces are the way to go indeed.

Reply:
Here we go: https://github.com/symfony/recipes/pull/1018 :)